example/operator/10-componentconfig.yaml

```diff
@@ -44,6 +44,8 @@ controllers:
         enableBackupCompaction: false
         eventsThreshold: 1000000
         activeDeadlineDuration: "3h"
+    # featureGates:
+    #   UseEtcdWrapper: true
```

Comment:
Either set this to `false` or remove it

Reply:
Actually, `true` should be ok since we usually follow the pattern `someKubernetesFeature: true` (see [here](https://github.com/gardener/gardener/blob/733e3d1b3242c5d3419ea34b38d610c6eb2e4f7c/example/90-shoot.yaml#L119))

Reply:
Sure, then we can leave it as is, since default behavior is `false` anyway